### PR TITLE
openshift-cli 3.7.0

### PR DESCRIPTION
**Pull request by acaire**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ilovezfs** [comment]
```
==> make all WHAT=cmd/oc GOFLAGS=-v OS_OUTPUT_GOPATH=1
hack/build-go.sh cmd/oc -v
[FATAL] Unable to find version for service catalog - (this should never happen)
[ERROR] PID 45904: hack/lib/build/binaries.sh:168: `version_ldflags=$(os::build::ldflags)` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/lib/build/binaries.sh:168: `version_ldflags=$(os::build::ldflags)`
[INFO] 		  2: hack/lib/build/binaries.sh:158: os::build::internal::build_binaries
[INFO] 		  3: hack/build-go.sh:43: os::build::build_binaries
[INFO]   Exiting with code 1.
[ERROR] hack/build-go.sh exited with code 1 after 00h 00m 01s
make: *** [all] Error 1
```

**Comment by acaire** [comment]
@ilovezfs fixed, thanks.

File: Formula/openshift-cli.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@ class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
   url "https://github.com/openshift/origin.git",
-    :tag => "v3.6.1",
-    :revision => "008f2d5528bf998326b5eb3f1fe3144c59392b9d"
+    :tag => "v3.7.0",
+    :revision => "7ed6862914ef20e22280c51199be5071e354999f",
+    :shallow => false
```
Comment (ilovezfs) [inline Formula/openshift-cli.rb]: why?

Comment (acaire) [reply Formula/openshift-cli.rb]: [empty var](https://github.com/openshift/origin/blob/v3.7.0/hack/lib/build/version.sh#L32)

Comment (ilovezfs) [reply Formula/openshift-cli.rb]: Ah OK!

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks again @acaire!

**Pull request closed without merging**